tests/e2e/specs/test.js

```diff
@@ -0,0 +1,8 @@
+// https://docs.cypress.io/api/introduction/api.html
+
+describe('My First Test', () => {
```

Comment:
should probably be adjusted before merge

Reply:
maybe we could merge https://github.com/aeternity/aepp-raendom/tree/feature/e2e-testing to this branch before we merge it to superhero

Reply:
good for me